yarn.lock

```diff
@@ -914,6 +914,11 @@
   resolved "https://registry.yarnpkg.com/@types/node/-/node-15.0.2.tgz#51e9c0920d1b45936ea04341aa3e2e58d339fb67"
   integrity sha512-p68+a+KoxpoB47015IeYZYRrdqMUcpbK8re/zpFB8Ld46LHC1lPEbp3EXgkEhAYEcPvjJF6ZO+869SQ0aH1dcA==
 
+"@types/node@^10.3.2":
```

Comment:
Thoughts on replacing `yarn` -> `npm`?

Reply:
No problem, let's totally replace yarn. I'll remove `yarn.lock` and update docs to use `npm` too